src/editor/components/modals/GeoModal/GeoModal.component.jsx

```diff
@@ -38,31 +40,33 @@ const GeoModal = ({ isOpen, onClose }) => {
       if (streetGeo && streetGeo['latitude'] && streetGeo['longitude']) {
         const lat = roundCoord(parseFloat(streetGeo['latitude']));
         const lng = roundCoord(parseFloat(streetGeo['longitude']));
-        const elevation = parseFloat(streetGeo['elevation']) || 0;
+        const ellipsoidalHeight = parseFloat(streetGeo['ellipsoidalHeight']);
 
         if (!isNaN(lat) && !isNaN(lng)) {
           setMarkerPosition({ lat, lng });
         }
-        if (!isNaN(elevation)) {
-          setElevation(elevation);
+        if (!isNaN(ellipsoidalHeight)) {
+          setElevation(ellipsoidalHeight);
         }
       }
     }
   }, [isOpen]);
 
-  const requestAndSetElevation = (lat, lng) => {
+  const requestAndSetElevation = async (lat, lng) => {
```

Comment:
If you don't use await in the function, then it's not async, even if your use then/catch.

Reply:
thanks i'll remove it